Release 2022.12.2 (#3852)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2022/index.md

## December

### Weekly release 2022.12.2

| Change | Expected in Preview Orgs |
|--------------------------------------------------------------------------|--------------------------|
| [Bugs fixed in 2022.12.2](#bugs-fixed-in-2022-12-2)                         |December 21, 2022            |

#### Bugs fixed in 2022.12.2

> * UPDATE requests to the `/brands/{brandId}/pages/sign-in/preview` and `/brands/{brandId}/pages/sign-in` endpoints sometimes returned a `null` response that resulted in an HTTP 400 Bad Request error. (OKTA-551632)

Reviewer: same comment from before

> * If an admin added redirect URIs that reached the maximum character limit, they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)

Reviewer: ```suggestion
* If an admin added a redirect URI that reached the character limit, then they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)
```
Minor update approved by @miaopan-okta .

> * If an admin added redirect URIs and reached the character limit, they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)

Reviewer: ```suggestion
* If an admin added redirect URIs that reached the maximum character limit, they couldn't edit the redirect URI list using the Apps API. (OKTA-476668)
```
same comment as above

### Weekly release 2022.12.1

| Change | Expected in Preview Orgs |
|--------------------------------------------------------------------------|--------------------------|

> | [Bugs fixed in 2022.12.1](#bugs-fixed-in-2022-12-1)                         |December 21, 2022            |

Reviewer: ```suggestion
| [Bug fixed in 2022.12.1](#bug-fixed-in-2022-12-1)                         |December 14, 2022            |
```

> | [Bugs fixed in 2022.12.2](#bugs-fixed-in-2022-12-2)                         |December 21, 2022            |

Author: error. fixed.

## December

### Weekly release 2022.12.2

| Change | Expected in Preview Orgs |
|--------------------------------------------------------------------------|--------------------------|
| [Bugs fixed in 2022.12.2](#bugs-fixed-in-2022-12-2)                         |December 21, 2022            |

#### Bugs fixed in 2022.12.2

> * PAttempts to save sign-in page edits sometimes failed when using the full-feature code editor. (OKTA-551632)

Reviewer: ```suggestion
* Attempts to save sign-in page edits sometimes failed when using the full-feature code editor. (OKTA-551632)
```
